libres/lib/include/ert/analysis/update.hpp

```diff
@@ -17,5 +17,15 @@ bool smoother_update(std::vector<int> step_list,
                      ensemble_config_type *ensemble_config,
                      enkf_fs_type *source_fs, enkf_fs_type *target_fs,
                      FILE *log_stream, bool verbose);
+void save_parameters(enkf_fs_type *target_fs,
+                     ensemble_config_type *ensemble_config,
+                     const int_vector_type *iens_active_index, int last_step,
+                     const local_ministep_type *ministep, matrix_type *A);
+
+matrix_type *load_parameters(enkf_fs_type *target_fs,
+                             ensemble_config_type *ensemble_config,
+                             const int_vector_type *iens_active_index,
+                             int last_step, int active_ens_size,
```

Comment:
The parameter `active_ens_size` is not required - that equals `int_vector_size( iens_active_index );`

Reply:
this does not work when only running a subset of realizations :shrug:

Reply:
OK I agree;
```C++
ens_size = int_vector_size( iens_active_index ) - int_vector_count_equal( iens_active_index, int_vector_get_default( iens_active_index));
```